packages/devtools_app/lib/src/ui/filter.dart

```diff
@@ -99,7 +97,7 @@ class _FilterDialogState<T> extends State<FilterDialog> with AutoDisposeMixin {
               const SizedBox(height: defaultSpacing),
             ],
             if (widget.toggleFilters != null)
-              for (final toggleFilter in widget.toggleFilters) ...[
+              for (final toggleFilter in widget.toggleFilters ?? []) ...[
```

Comment:
can you just use ! here instead since we already checked that widget.toggleFilters != null?

Reply:
done